app/backend/src/couchers/servicers/auth.py

```diff
@@ -208,11 +208,8 @@ def SignupFlow(self, request, context):
                     if not self._username_available(request.account.username):
                         context.abort(grpc.StatusCode.FAILED_PRECONDITION, errors.USERNAME_NOT_AVAILABLE)
 
-                    if request.account.password:
-                        abort_on_invalid_password(request.account.password, context)
-                        hashed_password = hash_password(request.account.password)
-                    else:
-                        hashed_password = None
+                    abort_on_invalid_password(request.account.password, context)
+                    hashed_password = hash_password(request.account.password)
```

Comment:
Remove the `// optional` markup from `SignupAccount::password` in `auth.proto` since it is now required.